TimberAPI/LocalizationSystem/LocalizationRepository.cs

```diff
@@ -94,10 +94,11 @@ private static IEnumerable<LocalizationRecord> TryToReadRecords(string localizat
         /// <returns></returns>
         private static List<string> GetLocalizationFilePathsFromDependencies(string localizationKey)
         {
+            PluginLocationService pluginLocationService = TimberAPI.DependencyContainer.GetInstance<PluginLocationService>();
```

Comment:
In these cases where the variables type is clear from the declaration, you could just write 
`var pluginLocationService = ...`
This is also recommended by [MS coding conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions), but I dunno do we care about that here

Reply:
We've been avoiding var (I'm allergic)